var/spack/repos/builtin/packages/coreneuron/package.py

```diff
@@ -96,8 +84,9 @@ class Coreneuron(CMakePackage):
     conflicts('+codegenopt', when='~nmodl')
     conflicts('+ispc', when='~nmodl')
 
-    # Caliper instrumentation is only supported after 1.0.1
-    conflicts('+caliper', when='@:1.0.1')
+    # Caliper instrumentation is only supported after 1.0.0.20210519
+    # Note: The 20210518 date is needed to specify a version before 20210519!
```

Comment:
This is a bit annoying, but I'm not immediately sure how we can achieve the same thing in a nicer way.

Reply:
It is definitely a bit odd, I agree. Perhaps we could implement our own version checking to support the `alpha`/`beta`/... subversions that Pramod suggested, but for now, this can work.